pipelines/docker-build-multi-platform-oci-ta/docker-build-multi-platform-oci-ta.yaml

```diff
@@ -311,7 +311,12 @@ spec:
       values:
       - "false"
     workspaces: []
-  - name: clamav-scan
+  - matrix:
+      params:
+      - name: image-arch
+        value:
+        - $(params.build-platforms)
```

Comment:
Ah, as caught by Andrew in https://github.com/konflux-ci/build-definitions/pull/2448#discussion_r2166647487, this won't really work